addon/snapshot.js

```diff
@@ -16,6 +16,27 @@ function getDoctype() {
   return doctype;
 }
 
+// Set the property value into the attribute value for snapshotting inputs
+function setAttributeValues(dom) {
+    // Limit scope to inputs only as textareas do not retain their value when cloned
+    let elems = dom.find('input[type=text], input[type=checkbox], input[type=radio]');
+
+    Ember.$(elems).each(function() {
+        switch(Ember.$(this).attr('type')) {
```

Comment:
An optimization: assign `$(this)` to a variable so that you don't have to reconstruct the element every time below.